code_schemes/dadaab_community_views_on_girls_education.json

```diff
@@ -3,6 +3,46 @@
   "Name": "DADAAB Community Views On Girls Education",
   "Version": "0.0.0.2",
   "Codes": [
+    {
+      "CodeID": "code-dfc56c39",
+      "CodeType": "Normal",
+      "DisplayText": "yes",
+      "NumericValue": 1,
+      "StringValue": "yes",
```

Comment:
Should this be "yes, other" given that there are other options for yes?

Reply:
This is for standalone yes/no responses. Happy to rename it to yes, no reason.

Reply:
I think that's better (yes, no reason), and is what we used to do to solve this ambiguity when we saw it in the past.

Reply:
updated with no_reason suffix